src/software/ai/passing/pass_generator.cpp

```diff
@@ -0,0 +1,174 @@
+#include "software/ai/passing/pass_generator.h"
+
+#include <iomanip>
```

Comment:
looks unused

Reply:
This is actually needed for `std::setprecision` (converting a floating point number to a string with fixed number of decimal points)